backend/oasst_backend/models/message_revision.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from typing import Optional
+from uuid import UUID, uuid4
```

Comment:
would it be possible to use uuid7?

Reply:
I used `uuid4` since all other tables in the backend are using that (but I noticed that the inference backend uses v7 strings). I could switch to using `uuid7str` for this table if that would be better. I just thought that it would be better if it was consistent.

Reply:
It is not about string vs not-string .. uuid7 has the right order for sorting by date (the uuid contains a timestamp). The storage format can simply be the classing UUID data type .. just the generator is different.

Reply:
(I am for using uuid7 everywhere in our backend)